roles/matrix-synapse/templates/ext/mautrix-telegram/systemd/matrix-mautrix-telegram.service.j2

```diff
@@ -21,6 +21,9 @@ ExecStart=/usr/bin/docker run --rm --name matrix-mautrix-telegram \
 			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
 			--cap-drop=ALL \
 			--network={{ matrix_docker_network }} \
+			{% if matrix_mautrix_telegram_container_expose_port %}
```

Comment:
If going with the `null` default value, it would be better if the check is more explicit (e.g. `{% if matrix_mautrix_telegram_container_exposed_port_number is not none %}`